docs/en/operation/Performance-Tuning.md

```diff
@@ -320,8 +318,8 @@ alluxio.master.persistence.blacklist=.staging,_temporary
 Files such as `/data/_temporary/part-00001`, `/data/temporary.staging` will not be considered for
 persist.
 This works because eventually these temporary files will be deleted or renamed to permanent files.
-Because `alluxio.user.file.persist.on.rename=true` is set, the files will be considered for persist
-again when renamed.
+Because `alluxio.user.file.persist.on.rename=true` is set, the files will be considered for
+persistence again when renamed.
 Note that persist on rename works for directories as well as files - if a top level directory is
 renamed with the persist on rename option, all its not yet persisted files will be scanned for
-eligbility for persist.
+eligibility for persist.
```

Comment:
"...will be scanned for eligibility for persist" sounds kind of odd.

How do you feel about: "any files underneath the top-level directory will be considered for persistence"
or
"any files underneath the top-level directory will be considered for persist eligibility"

Reply:
updated